apps/taiga-lumbermill/src/dashboards/crypto/components/minter/minter-deploy/minter-deploy.component.html

```diff
@@ -0,0 +1,84 @@
+@let token = minterForm.controls[0].value;
+@let symbol = minterForm.controls[1].value;
+@let amount = minterForm.controls[2].value;
+@let description = minterForm.controls[3].value;
+<div class="info">
+    <div tuiCell="l">
+        <tui-avatar
+            size="xl"
+            [src]="!openIcon && urlIcon !== '' ? urlIcon : '@tui.pencil'"
+            (click)="showDialog()"
+        />
+        <div tuiTitle>
+            {{ token == '' ? 'Jetton Name' : token }} ({{ symbol == '' ? 'Symbol' : symbol }})
+            <div tuiSubtitle>{{ description == '' ? 'Description' : description }}</div>
+        </div>
+    </div>
+</div>
+@for (control of minterForm['controls']; track $index) {
+    <div class="block-input">
```

Comment:
we should completely redesign this form without the cycle

Reply:
just remove the loop or change the logic altogether?

Reply:
it should be plain classic form
like 
```
<form [formGroup]="formGroup">
   <input formControlName="name">
    <tui-error
            formControlName="name"
            [error]="[] | tuiFieldError | async"
        />

      <input formControlName="name">
    <tui-error
            formControlName="name"
            [error]="[] | tuiFieldError | async"
        />
... 
```